variables.tf

```diff
@@ -1,5 +1,100 @@
-variable "length" {
-  description = "The length of the random name"
-  type        = number
-  default     = 2
+variable "github_organization" {
+  type        = string
+  description = "The GitHub organization name"
```

Comment:
Given https://github.com/integrations/terraform-provider-github/pull/241, I wonder if we should add that `github_organization` is case insensitive?

Reply:
Hmm, I think this is fixed. I don't see drifts in `mp-infra` root module, and we have there various cases.
See https://github.com/masterpointio/mp-infra/pull/60